metadata-ingestion/src/datahub/ingestion/source/tableau/codegen_turms/schema.py

```diff
@@ -0,0 +1,1893 @@
+    BaseModel,
+):
+    typename: Literal["DataCloud"] = Field(alias="__typename", default="DataCloud")
+
+
+class GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseDatabaseServer(
+    GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBase,
+    BaseModel,
+):
+    typename: Literal["DatabaseServer"] = Field(
+        alias="__typename", default="DatabaseServer"
+    )
+
+
+class GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseFile(
+    GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBase,
+    BaseModel,
+):
+    typename: Literal["File"] = Field(alias="__typename", default="File")
+
+
+class GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseWebDataConnector(
+    GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBase,
+    BaseModel,
+):
+    typename: Literal["WebDataConnector"] = Field(
+        alias="__typename", default="WebDataConnector"
+    )
+
+
+class GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtables(
+    BaseModel
+):
+    """A table that is contained in a database"""
+
+    typename: Literal["DatabaseTable"] = Field(
+        alias="__typename", default="DatabaseTable"
+    )
+    id: str
+    "Unique identifier used by the metadata API.  Not the same as the numeric ID used on server"
+    name: Optional[str] = Field(default=None)
+    "Name shown in server and desktop clients"
+    database: Optional[
+        Annotated[
+            Union[
+                GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseCloudFile,
+                GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseDataCloud,
+                GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseDatabaseServer,
+                GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseFile,
+                GetItems_customSQLTablesConnectionCustomsqltablesconnectionNodesColumnsReferencedbyfieldsDatasourceUpstreamtablesDatabaseBaseWebDataConnector,
+            ],
+            Field(discriminator="typename"),
+        ]
+    ] = Field(default=None)
+    "The database to which this table belongs"
+    the_schema: Optional[str] = Field(
+        default=None, alias="schema"
+    )  # !!! schema is a reserved word
```

Comment:
I haven't found a way to tell turms `schema` is a reserved keyword or that an alias should be defined.

This is definitely a red flag 🟥